libdebug/cffi/ptrace_cffi_source.c

```diff
@@ -339,6 +693,56 @@ uint64_t ptrace_pokeuser(int pid, uint64_t addr, uint64_t data)
 {
     return ptrace(PTRACE_POKEUSER, pid, addr, data);
 }
+#endif
+
+#ifdef ARCH_AARCH64
+// peekuser doesn't exist on ARM64, we have to use getregset
```

Comment:
did you try PEEKUSR / POKEUSR ?

Reply:
Where does it come from? I have never heard about it

Reply:
In some ptrace file PEEKUSER is defined as PEEKUSR and some ODC students told me that PEEKUSR is exposed on ubuntu for arm. I did not check, actually. Was just a suggestion